src/main/java/com/synopsys/integration/detect/lifecycle/run/operation/OperationFactory.java

```diff
@@ -750,6 +758,13 @@ public HttpUrl findProjectGroup(BlackDuckRunData blackDuckRunData, String projec
         });
     }
 
+    public List<String> findLicenseUrls(BlackDuckRunData blackDuckRunData, List<String> licenseNames) throws DetectUserFriendlyException {
+        return auditLog.namedInternal("Find License Urls By Name", () -> {
```

Comment:
```suggestion
        return auditLog.namedInternal("Find License Urls By Name", "LicenseUrlLookup", () -> {
```
Let's try adding phone home tags to new operations